go.mod

```diff
@@ -35,7 +39,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.8.4
-	golang.org/x/crypto v0.14.0
+	golang.org/x/crypto v0.17.0
```

Comment:
Hey Iramis, why is this version being bumped?

Reply:
Seems like the new hashicorp deps rely on this version

Reply:
Discussed separately, resolved